calblueprint/siren-web
# User Auth

# ✨ New in this PR ✨
User auth completed! /welcome, /login, /register urls

Note: this branch is a subbranch of andrei's pr
## One liner: what did you do? 🚨 
completed user auth
## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] Welcome
 - [X] Log in
 - [X] Register
 - [] Forgot password functionality
 - [] View password functionality
 - [] Error handling

## How can the reviewer test your code? 👩‍💻
_example: run `npm start` and click through Navigation bar_ 
run `npm run dev` and go to local host, it should redirect you to the welcome page. Register and then login, view firebase auth users to make sure your user was created successfully.

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 


## Resources 📔
_link online resources and related PRs_ 


🧜‍♀️ cc: @shannonbonet @phoebeli23
